### PR TITLE
[docs][filebeat]: add 4th step in migrate-to-filestream

### PR DESCRIPTION
**Pull request by VihasMakwana**
## Proposed commit message
Follow up PR from https://github.com/elastic/beats/pull/39828.

Add a 4th step to enure that the user disables take_over mode once they start receiving events with `take_over` tag.
## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: filebeat/docs/howto/migrate-to-filestream.asciidoc [hunk 1]
```diff
@@ -234,6 +234,13 @@ The example configuration shown earlier needs to be adjusted as well:
 |backoff.max
 |===
 
+=== Step 4
+
+The events produced by `filestream` input conatains `take_over` tag.
```
Comment (pierrehilbert) [inline filebeat/docs/howto/migrate-to-filestream.asciidoc:239]:
```suggestion
The events produced by `filestream` input contains `take_over` tag.
```

File: filebeat/docs/howto/migrate-to-filestream.asciidoc [hunk 2]
```diff
@@ -234,6 +234,13 @@ The example configuration shown earlier needs to be adjusted as well:
...
+You can filter on this tag in Kibana and see the events which came from a filestream in the "take over" mode.
```
Comment (pierrehilbert) [inline filebeat/docs/howto/migrate-to-filestream.asciidoc:240]:
```suggestion
You can filter on this tag in Kibana and see the events coming from a filestream in the "take_over" mode. 
```

**Review by jlind23 — APPROVED** [review]

**Review by pierrehilbert — APPROVED** [review]

**Comment by pierrehilbert** [comment]
@VihasMakwana even if Julien & I approved the PR, please wait for @kilfoyle's review to ensure consistency between our docs.

File: filebeat/docs/howto/migrate-to-filestream.asciidoc [hunk 3]
```diff
@@ -234,6 +234,13 @@ The example configuration shown earlier needs to be adjusted as well:
 |backoff.max
 |===
 
+=== Step 4
+
+The events produced by `filestream` input with `take_over: true` contain `take_over` tag.
+You can filter on this tag in Kibana and see the events which came from a filestream in the "take_over" mode. 
+
+Once you start receiving event with this tag, you can remove `take_over: true` and restart the fileinput again.
```
Comment (kilfoyle) [inline filebeat/docs/howto/migrate-to-filestream.asciidoc:242]:
```suggestion
The events produced by `filestream` input with `take_over: true` contain a `take_over` tag.
You can filter on this tag in Kibana and see the events which came from a filestream in the "take_over" mode. 

Once you start receiving events with this tag, you can remove `take_over: true` and restart the fileinput again.
```

**Review by kilfoyle — APPROVED** [review]
LGTM! 🚀 
Just one small suggestion.

**Pull request merged**